go.mod

```diff
@@ -14,14 +17,19 @@ require (
 )
 
 require (
+	github.com/antlr4-go/antlr/v4 v4.13.0 // indirect
```

Comment:
Weird that these weren't deps from locally generated code before :/

Reply:
I think these are transitive from the protovalidate deps.